README.md

```diff
@@ -154,6 +154,7 @@ See the [README](doc/README.md) file in the doc directory for more info.
   - [BetterTouchTool](http://blog.boastr.net/)
   - [BibDesk](http://bibdesk.sourceforge.net/)
   - [Billings Pro Server Admin](https://www.marketcircle.com/billingspro/download/billingspro-server/)
+  - [Bitchx](https://github.com/BitchX/)
```

Comment:
Ok, I will clean it up.

On Friday, April 17, 2015, Laurent Raufaste <notifications@github.com>
wrote:

> In README.md <https://github.com/lra/mackup/pull/516#discussion_r28635382>
> :
>
> > @@ -154,6 +154,7 @@ See the [README](doc/README.md) file in the doc directory for more info.
> >    - [BetterTouchTool](http://blog.boastr.net/)
> >    - [BibDesk](http://bibdesk.sourceforge.net/)
> >    - [Billings Pro Server Admin](https://www.marketcircle.com/billingspro/download/billingspro-server/)
> > +  - [Bitchx](https://github.com/BitchX/)
>
> I don't mind which one, but it should be the official one.
> https://github.com/BitchX/ looks less updated than http://www.bitchx.org/
> It's a mess =)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/lra/mackup/pull/516/files#r28635382>.
>


-- 
Troy Wilson
Phone: 913.667.9662
Fax: 1.877.748.9636
Telegram Messenger: @Troy_Wilson
Troy@MerchantGuy.com

PGP Signed Emails and PGP Encryption available upon request.